test/e2e/performanceprofile/functests/0_config/config.go

```diff
@@ -44,6 +45,24 @@ var _ = Describe("[performance][config] Performance configuration", func() {
 		RunningOnSingleNode = isSNO
 	})
 
+	It("should remove OLM artifacts for performance-addon-operator", func() {
```

Comment:
I'm trying to think if we should have e2e tests for both the scenarios:

1. <= 4.10 PAO installed
2. no PAO installed at all
and I'm inclined to think yes.

However we can start with integration tests to cover all the flows, like we had previously

Reply:
This test case is pretty general but it will cover the basics of 1 and 2. 
But now thinking of it the test might need to be fixed to get matching subscriptions first according to the new code.

Reply:
ok, I guess we can revisit later if needed